src/psyclone/tests/domain/common/extract_driver_creator_test.py

```diff
@@ -373,16 +391,17 @@ def test_driver_creation_create_flattened_symbol_errors(monkeypatch):
     monkeypatch.setitem(grid_properties, "go_grid_lat_u", prop)
     # We need a new instance, since the instance above was modified
     edc = ExtractDriverCreator()
-    with pytest.raises(TransformationError) as err:
+    with pytest.raises(InternalError) as err:
         edc.create_flattened_symbol("new_name", ref, SymbolTable())
-    assert "Unknown gocean property type 'invalid-type' in expression " \
-           "'in_fld%grid%gphiu." in str(err.value)
+    assert re.search("The expression 'in_fld%grid%gphiu' maps to an unknown "
+                     "GOcean property type 'invalid-type' in the config "
+                     "file '.*'.", str(err.value))
 
 
 # -----------------------------------------------------------------------------
 def test_errors_add_call():
-    '''Test that it is detected if a call added that is not a
-    RoutineSymbol.
+    '''Test than an error is raised if the symbol supplied to 'add_call()' is
```

Comment:
You've cut-n-pasted my own typo :-) s/than/that/

Reply:
I apparently trust your every word.